get_class_matched_articles.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import re
+import shutil
+import requests
+import sys
+
+class DBpediaSearcher:
```

Comment:
I think that using object-oriented programming is over-engineering here.
We just need some functions and that's it